backend-rs/src/common.rs

```diff
@@ -24,11 +25,18 @@ impl From<sqlx::Error> for AppError {
     }
 }
 
+impl From<anyhow::Error> for AppError {
+    fn from(value: anyhow::Error) -> Self {
+        AppError::UnexpectedError(value)
+    }
+}
+
 impl IntoResponse for AppError {
     fn into_response(self) -> axum::response::Response {
         let body = match self {
             AppError::OpenSearchError(error) => error.to_string(),
             AppError::SqlxError(error) => error.to_string(),
+            AppError::UnexpectedError(error) => error.to_string(),
         };
         (StatusCode::INTERNAL_SERVER_ERROR, Json(body)).into_response()
```

Comment:
@Centeno448 If all of these AppError types are marked as "INTERNAL_SERVER_ERROR" here, we should be able to merge them if we do not do anything special with each of them.